src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -21,6 +21,10 @@ export default async function noUnusedAndMissingDependencies() {
     // Sass (eg. in create-react-app)
     'sass',
 
+    // Stylelint configuration
+    'stylelint-config-css-modules',
+    'stylelint-config-recommended-scss',
```

Comment:
Are these the only dependencies that we need to ignore? Or are there more? (eg. Stylelint deps, Styled Components deps, PostCSS deps)?

Reply:
i add a new one needed

Reply:
[a496b22](https://github.com/upleveled/preflight/pull/346/commits/a496b22a81a1c1453c80c422595609f1f2b55414)

Reply:
ok yeah no problem if we don't need these other things, just test out `create-react-app` and `create-next-app` (along with adding the ESLint config) and if those work, then that's good enough for now

students will report it anyway if it breaks